.circleci/config.yml

```diff
@@ -34,7 +34,8 @@ jobs:
           command: |
             source dev_env.sh
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
-            pip install psycopg2==2.8.6
+            apt install -y libpq-dev
+            pip install psycopg2==2.8.4
```

Comment:
This is worth providing some context. Tap-tester is running python 3.9 but the tap is on 3.5.6.  

3.9 is incompatible with psycopg2==2.7.4 so it needs psycopg2==2.8.4 (tap-tester uses psycopg2 to set up the db in the tests)

3.5.6 is incompatible with psycopg2==2.8.4 so it needs psycopg2==2.7.4